webpack/assets/javascripts/react_app/pages/AuditsPage/__snapshots__/AuditsPage.test.js.snap

```diff
@@ -0,0 +1,165 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`render audits page 1`] = `
+<PageLayout
+  breadcrumbOptions={null}
+  customBreadcrumbs={null}
+  header="Audits"
+  searchProps={Object {}}
+  searchable={true}
+  toastNotifications=""
+  toolbarButtons={
+    <Button
+      active={false}
+      block={false}
+      bsClass="btn"
+      bsStyle="default"
+      className="btn-docs"
+      disabled={false}
+      href="http://www.theforeman.org/manuals/1.21/index.html#4.1.4Auditing"
```

Comment:
This means we'll need to update snaps on every version bump. I'd like to avoid this but if needed, it must be documentation in the release process. cc @tbrisker